### ISSUE TITLE
Rails/BulkChangeTable fails due to ERB in database.yml

### ISSUE THREAD

**Issue by haines** [rubocop-hq/rubocop#5987]
If there is ERB in database.yml that causes it to not be syntactically valid YAML without pre-processing, the Rails/BulkChangeTable cop fails.

For example, we have a ternary in one of our keys, which is not valid yaml because the colon is a special character:
```yaml
  pool: <%= Sidekiq.server? ? 10 : 5 %>
```

--------

## Expected behavior

Rubocop not to throw an error, and to suggest setting the `Database` config for this cop.

## Actual behavior

Rubocop throws an error:

```
(config/database.yml): mapping values are not allowed in this context at line 44 column 34
psych.rb:377:in `parse'
# ...
$GEM_HOME/gems/rubocop-0.57.1/lib/rubocop/cop/rails/bulk_change_table.rb:194:in `database_yaml'
```

## Steps to reproduce the problem

Add a ternary inside an ERB expression in database.yml, then run the Rails/BulkChangeTable cop.

## RuboCop version

```
$ rubocop -V
0.57.1 (using Parser 2.5.1.0, running on ruby 2.3.0 x86_64-darwin17)
```
